### DISCUSSION TITLE
conference materials

### DISCUSSION THREAD

**Discussion by pelikhan** [gatsbyjs/gatsby#29985] [Help]
Is there a list of URLs of the tools mentionned in "The Anatomy of Performance Audit" ?

**Comment by calcsam** [comment]
Thanks for asking! Here are some links:
* video at https://www.youtube.com/watch?v=x4RfrJiIzBE 
* docs at https://gatsby.dev/site-performance
* All tools are listed in the docs should be findable by Googling (if they aren't already linked(. The only one that isn't is `gatsby-plugin-perf-budgets` -- you can find docs there athttps://www.npmjs.com/package/gatsby-plugin-perf-budgets
